app/models/dataset.rb

```diff
@@ -0,0 +1,19 @@
+class Dataset < ActiveRecord::Base
+
+  #relationships
+  belongs_to :creator, class_name: 'User', foreign_key: 'creator_id', inverse_of: :created_datasets
+  has_many :dataset_items
+
+  # add deleted_at and deleter_id
+  # TODO: ask anthony
```

Comment:
convert this comment to something along the lines of:

> We haven't enabled soft deletes yet since we do not support deleting datasets. We may change this in the future.